frontend/README.md

```diff
@@ -7,7 +7,7 @@ Please also read the [guidance](../README.md#guidance) and instructions for [sub
 
 ## Task
 
-Write a frontend application in TypeScript that Fetches and displays
+Write a frontend application in TypeScript that fetches (via a HTTP request) and displays
```

Comment:
If they are reading a local file it's not going to be a HTTP req. Having said that, maybe we should deploy the data to a lambda or a cloudflare worker so they can work with it?

Reply:
The data can be retrieved via HTTP... https://raw.githubusercontent.com/AssetzSMECapital/technical-challenge/master/data/holdings.json but if it's still not clear enough, we perhaps need to clarify that further.

I was hoping to keep everything self-contained, rather than host part of the solution elsewhere.